src/steps/11-GeneratePackageDocuments/components/ReviewDocuments.vue

```diff
@@ -124,59 +152,149 @@ import FinancialDetails from "@/store/financialDetails";
     ATATAlert
   }
 })
-export default class ReviewDocuments extends Vue {
 
+export default class ReviewDocuments extends Vue {
   @PropSync(
     "packageDocuments",{default: () => []}
   ) private _packageDocuments!: [];
-
   @Prop({default: false }) private isErrored!: boolean;
-  public packageId = "";
-
 
-  private contractingShop = "";
+  public packageId = "";
+  private lastUpdatedString = ""
+  private currentEnvServiceName = CURRENT_ENVIRONMENT_TABLE;
+  private needsSignatureLength = 0
   get fairOpportunity():string {
     return AcquisitionPackage.fairOpportunity?.exception_to_fair_opportunity || "";
   }
   get incrementallyFunded():string {
     return FinancialDetails.fundingRequirement?.incrementally_funded || "";
   }
-  private packageCheckList: Record<string,string|boolean>[] = []
+  get ditcoUser():boolean {
+    return AcquisitionPackage.acquisitionPackage?.contracting_shop === "DITCO"
+  }
```

Comment:
- [ ] Duplicate code. if possible, put this in the Acq package store as it is used a couple of times.